src/App.jsx

```diff
@@ -14,57 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === '' && (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood !== '' && `${selectedGood} is selected`}
+        {selectedGood !== '' && (
+          <button
+            onClick={() => setSelectedGood('')}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        )}
+      </h1>
```

Comment:
You have two separate `<h1>` elements for displaying the selection status. This can be simplified into a single `<h1>` with conditional rendering inside. This approach avoids rendering an empty `<h1>` tag and makes the code cleaner.

```suggestion
<h1 className="title is-flex is-align-items-center">
  {selectedGood ? (
    <>
      {`${selectedGood} is selected`}
      <button
        onClick={() => setSelectedGood('')}
        data-cy="ClearButton"
        type="button"
        className="delete ml-3"
      />
    </>
  ) : (
    'No goods selected'
  )}
</h1>
```